app/kuma-cp/cmd/root.go

```diff
@@ -41,6 +50,10 @@ func newRootCmd() *cobra.Command {
 	}
 	// root flags
 	cmd.PersistentFlags().StringVar(&args.logLevel, "log-level", kuma_log.InfoLevel.String(), kuma_cmd.UsageOptions("log level", kuma_log.OffLevel, kuma_log.InfoLevel, kuma_log.DebugLevel))
+	cmd.PersistentFlags().StringVar(&args.outputPath, "output-path", args.outputPath, "path of output rotating file")
```

Comment:
so let say I pass `/tmp/kuma.log` what happens when the file gets rotated?

Reply:
Ok, I tested it and the result is
```
❯❯❯ ls -lah /tmp/ | grep kuma
-rw-r--r--   1 jakob  wheel   1.0M Jun  7 11:15 kuma-2021-06-07T09-15-18.265.log
-rw-r--r--   1 jakob  wheel   1.0M Jun  7 11:15 kuma-2021-06-07T09-15-58.193.log
-rw-r--r--   1 jakob  wheel   206K Jun  7 11:16 kuma.log
```

Reply:
I think in the help I'd write `path of the rotating log file. Example: if we set it to /tmp/kuma.log then after the file is rotated we will have /tmp/kuma-2021-06-07T09-15-18.265.log`